EmpireLegends/GameEngine.cpp

```diff
@@ -15,8 +15,9 @@ GameEngine::GameEngine()
 GameEngine::~GameEngine()
 {
 	delete gameStart;
+	//Conflict between startPhase & mainGame when closing  bc they both delete a map
 	delete startPhase;
-	delete mainGame;
+	//delete mainGame;
```

Comment:
You can't leave this commented out. Also, I'm assuming this would cause a memory leak

Reply:
Resolved it in new update